integrations/huggingface-transformers/tutorials/text-classification/text-classification-cli.md

```diff
@@ -0,0 +1,583 @@
+
+sick = load_dataset("sick")
+print(sick)
+print(sick["train"][0])
+sick["train"].to_csv("./sick-train.csv")
+sick["validation"].to_csv("./sick-validation.csv")
+```
+
+We can see that the data is a CSV file with `sentence_A` and `sentence_B` as the input columns and `label` as the label column:
+
+```bash
+head ./sick-train.csv --lines=5
+```
+
+#### **Kick off Training**
+
+To use the local files with the CLI, pass `--train_file ./sick-train.csv --validation_file ./sick-validation.csv  --input_column_names 'sentence_A,sentence_B' --label_column_name label`.
+
+Run the following:
+```bash
+sparseml.transformers.text_classification \
+  --model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none \
+  --recipe zoo:nlp/text_classification/obert-base/pytorch/huggingface/mnli/pruned90_quant-none \
+  --distill_teacher disable \
+  --train_file ./sick-train.csv --validation_file ./sick-validation.csv --input_column_names 'sentence_A,sentence_B' --label_column_name 'label' \
+  --output_dir sparse_quantized_bert-text_classification_sick-csv \
+  --do_train --do_eval --max_seq_length 128 --evaluation_strategy epoch --logging_steps 1000 --save_steps 1000 \
+  --per_device_train_batch_size 8 --per_device_eval_batch_size 32 --gradient_accumulation_steps 4 --preprocessing_num_workers 32 \
+  --seed 5114
+```
+
+### Sparse Transfer Learning with a Custom Teacher (Rotten Tomatoes)
+
+To support the transfer learning process, we can apply model distillation, just like we did for the GLUE task case.
+You are free to use the native Hugging Face workflows to train the dense teacher model (and can even
+pass a Hugging Face model identifier to the command), but you can also use the SparseML CLI as well. 
+
+#### Train The Dense Teacher
+
+Run the following to train a dense model on Rotten Tomatoes:
+```
+sparseml.transformers.train.text_classification \
+  --output_dir dense-teacher_rotten_tomatoes \
+  --model_name_or_path zoo:nlp/masked_language_modeling/bert-base/pytorch/huggingface/wikipedia_bookcorpus/base-none \
+  --recipe zoo:nlp/sentiment_analysis/bert-base/pytorch/huggingface/sst2/base-none \
+  --recipe_args '{"init_lr":0.00003}' \
+  --dataset_name rotten_tomatoes --input_column_names "text" --label_column_name "label" \
+  --max_seq_length 128 --per_device_train_batch_size 32 --per_device_eval_batch_size 32 --preprocessing_num_workers 6 \
+  --do_train --do_eval --evaluation_strategy epoch --fp16  \
+  --save_strategy epoch --save_total_limit 1
+```
+
+Note that used the dense version of BERT (the stub ends in `base-none`) as the starting point for the training 
+and passed a recipe from [SparseZoo](https://sparsezoo.neuralmagic.com/models/nlp%2Fsentiment_analysis%2Fbert-base%2Fpytorch%2Fhuggingface%2Fsst2%2Fbase-none) which was used to train the 
+dense teacher for the SST2 task. Since the SST2 task is similiar to the Rotten Tomatoes task, these parameters are a solid
```

Comment:
Since the SST2 task is similar to the Rotten Tomatoes task, these parameters are a solid place to start